examples/learn/prompts/multi_modal/audio/messages.py

```diff
@@ -1 +1,23 @@
-# Coming soon...
+import wave
+from mirascope.core import prompt_template, Messages
+
+
+@prompt_template()
+def identify_book_prompt(audio_wav: wave.Wave_read) -> Messages.Type:
```

Comment:
what about shorthand for non-wave audio files? like mp3?

it seems like we should either use `pydub.AudioSegment` or we should add additional tabs showing both `pydub.AudioSegment` and `wave.Wave_read`?